master/internal/api_tasks.go

```diff
@@ -143,35 +144,56 @@ func (a *apiServer) TaskLogs(
 func (a *apiServer) GetTasksCount(
 	_ context.Context, req *apiv1.GetTasksCountRequest,
 ) (resp *apiv1.GetTasksCountResponse, err error) {
-	finalResp := &apiv1.GetTasksCountResponse{}
+	finalResp := &apiv1.GetTasksCountResponse{
```

Comment:
These don't need to be explicitly initialized, and I'd say it's more idiomatic to not do so.